docs/howto-deploy-modules.md

```diff
@@ -7,10 +7,10 @@ There are several options to deploy modules to your [Azure IoT Edge](https://azu
 
 For concrete examples as to how to deploy the Azure Industrial IoT Edge Modules follow the individual module deployment guides for each of the Azure Industrial IoT Services or modules:
 
-- [Deploy OPC UA Device Management modules](twin/howto-modules.md)
+- [Deploy OPC UA Twin and Publisher modules](twin/howto-modules.md)
 - Deploy OPC UA Certificate Management module (COMING SOON).
```

Comment:
While you are at it, there was an ask to make sure we are consistent in terminology and naming:  Could you please.
- change OPC UA Device Management or OPC UA Twin or OPC UA Publisher to OPC Twin and OPC Publisher respectively.  
- change "OPC UA Certificate Management" to "OPC Vault" wherever